src/sqlparser/tests/testdata/set.yaml

```diff
@@ -0,0 +1,17 @@
+# This file is automatically generated. See `src/sqlparser/test_runner/src/bin/apply.rs` for more information.
+- input: SET TIME ZONE LOCAL
+  formatted_sql: SET TIME ZONE LOCAL
+- input: SET TIME ZONE DEFAULT
+  formatted_sql: SET TIME ZONE DEFAULT
+- input: SET TIME ZONE "Asia/Shanghai"
+  formatted_sql: SET TIME ZONE "Asia/Shanghai"
+- input: SET TIME ZONE 'Asia/Shanghai'
```

Comment:
The error is unexpected, but I don't want to fix it now. Will submit an issue after the PR got merged.